docs/README.md

```diff
@@ -8,7 +8,7 @@
 
 # Kotlin kernel for IPython/Jupyter
 
-[Kotlin](https://kotlinlang.org/) (1.5.30-dev-2630) kernel for [Jupyter](https://jupyter.org).
+[Kotlin](https://kotlinlang.org/) (1.5.20) kernel for [Jupyter](https://jupyter.org).
```

Comment:
Shouldn't be changed by regeneration, I suppose. Please, rebase on master and use default Kotlin version, dependencies problems should be fixed now